mypy/util.py

```diff
@@ -661,12 +660,12 @@ def initialize_win_colors(self) -> bool:
                 | ENABLE_VIRTUAL_TERMINAL_PROCESSING,
             )
             self.initialize_vt100_colors()
-            return True
-        return False
+        return True
 
     def initialize_unix_colors(self) -> bool:
         """Return True if initialization was successful and we can use colors, False otherwise"""
-        if sys.platform == "win32" or not CURSES_ENABLED:
+        is_win = sys.platform == "win32"
+        if is_win or not CURSES_ENABLED:
             return False
```

Comment:
This can also be solved by indenting the `try` in an `else`. (or putting a `# type: ignore[unreachable]` with a comment linking to https://github.com/python/mypy/issues/10773 so that we have an explicit marker that the work has been deferred)

Reply:
Yeah, `type: ignore` can be used here, since this is a bug. Thanks!

Reply:
Hm, on the other hand: `[unreachable]` won't be reported on Linux builds. So, I think we can keep the current solution. I also don't quite like the current hack :(

Indenting `try` will also create more diff lines, which I try to avoid during refactorings.

Reply:
`# type: ignore[unreachable,unused-ignore]` might work but yeah that's a bit much.